components/prism-bqn.js

```diff
@@ -0,0 +1,65 @@
+Prism.languages.bqn = {
+	'shebang': {
+		pattern: /^#![ \t]*\/.*/,
+		alias: 'important',
+		greedy: true
+	},
+	'comment': {
+		pattern: /#.*/,
+		greedy: true
+	},
+	'string-literal': {
+		pattern: /"(?:\\[\s\S]|\$(?!\()|`[^`]+`|[^"\\`$])*"/,
```

Comment:
BQNs string literals are way simpler - `/"(?:[^"]|"")*"/`

Reply:
Oh, thanks! The one I was using was from the APL highlighting, but I was having difficulty massaging it to recognized the double-double quote escape. This fixed it! Thanks again.